src/main/index.ts

```diff
@@ -61,20 +63,24 @@ if (process.env.LENS_DISABLE_GPU) {
 
 app.on("ready", async () => {
   logger.info(`🚀 Starting Lens from "${workingDir}"`);
+  logger.info("🐚 Syncing shell environment");
   await shellSync();
 
   powerMonitor.on("shutdown", () => {
     app.exit();
   });
 
+  logger.info(`📡 Checking for app updates`);
   const updater = new AppUpdater();
 
   updater.start();
 
   registerFileProtocol("static", __static);
 
+  logger.info("🤓 Installing developer tools");
```

Comment:
I think we skip these on production build? Logging this on production is probably wrong?

Reply:
@nevalla ping

Reply:
Yes, it's wrong. Moved the log line inside the installation method.